packages/account/src/Sections/Security/Passwords/__tests__/deriv-password.spec.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { render, screen, waitFor, fireEvent } from '@testing-library/react';
+import DerivPassword from '../deriv-password';
+import { WS } from '@deriv/shared';
+
+jest.mock('Assets/ic-brand-deriv-red.svg', () => () => 'BrandDerivRed');
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    WS: {
+        verifyEmail: jest.fn(),
+    },
+}));
+
+describe('<DerivPassword />', () => {
+    let mock_props = {
+        email: 'mf@deriv.com',
+        is_social_signup: false,
+        social_identity_provider: undefined,
+        landing_company_shortcode: 'maltainvest',
+    };
+
+    let modal_root_el;
+
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+    test('Should render properly', async () => {
```

Comment:
can we use `it` instead of `test`. We use `it` in out test cases everywhere :pray: